streamer/src/test/e2e/tests.js

```diff
@@ -0,0 +1,107 @@
+        equal(missedValidators.count, 0,
+            `There is discrepancy between sum of validators_active in ${environment.DB_SCHEMA}.eras and count ` +
+                    `of validators in ${environment.DB_SCHEMA}.validators for corresponding value of era; Eras : ${missedValidators.eras}`
+        )
+    })
+
+    it('Check table eras for misses by nominatorsCount in nominators table', async () => {
+        const missedNominators = await postgresQueries.getMissedNominators(postgres)
+        equal(missedNominators.count, 0,
+            `There is discrepancy between sum of nominators_active in ${environment.DB_SCHEMA}.eras and count ` +
+                    `of nominators in ${environment.DB_SCHEMA}.nominators for corresponding value of era; Eras : ${missedNominators.eras}`
+        )
+    })
+
+    it('Check missed hashes for sequence of blocks', async () => {
+        const missedHashes = await postgresQueries.getMissedHashesForSequenceOfBlocks(postgres)
+        equal(missedHashes.breakCount, 0,
+            `There is ${missedHashes.breakCount} hashes sequence breaks; ` +
+                    `Breaks info: ${JSON.stringify(missedHashes.info)}`
+        )
+    })
+
+    it('Check node top block  with streamer top block', async () => {
+        const topBlock = await postgresQueries.getTopBlock(postgres)
+        const nodeTopBlock = await nodeApi.rpc.chain.getHeader()
+        const topBlockDifference = nodeTopBlock.number.toString() - topBlock
+        equal(topBlockDifference < 100, true,
+            `Streamer top block is lag behind of node top block for ${topBlockDifference} blocks`)
+    })
+
+    it('Check eras for low total stake', async () => {
+        const erasCountWithLowTotalStake = await postgresQueries.getErasWithLowTotalStack(postgres)
+        equal(erasCountWithLowTotalStake.erasCount, 0,
+            `There is ${erasCountWithLowTotalStake.eras} eras with low total stake`)
+    })
+
+    it('Check eras for low total reward', async () => {
+        const erasCountWithLowTotalReward = await postgresQueries.getErasWithLowTotalReward(postgres)
+        equal(erasCountWithLowTotalReward.erasCount, 0,
+            `There is ${erasCountWithLowTotalReward.eras} eras with low total rewards`)
+    })
+
+    it('Check eras for low total reward points', async () => {
+        const erasCountWithLowTotalRewardPoints = await postgresQueries.getErasWithLowTotalRewardPoints(postgres)
+        equal(erasCountWithLowTotalRewardPoints.erasCount, 0,
+            `There is ${erasCountWithLowTotalRewardPoints.eras} eras with low total rewards points`)
+    })
+
+    it('Check rewards points in eras and sum of reward points in validators by eras', async () => {
+        const rewardPointsInconsistenciesEras = await postgresQueries.getErasWithRewardPointsInconsistencies(postgres)
+        equal(JSON.stringify(rewardPointsInconsistenciesEras.erasCount), 0,
+            `There is ${JSON.stringify(rewardPointsInconsistenciesEras.erasCount)} ` +
```

Comment:
Why JSON.stringify used here? 
Looks like rewardPointsInconsistenciesEras.erasCount has Number type

Reply:
Yes,rewardPointsInconsistenciesEras.erasCount has a Number type and JSON.stringify shouldn't be used here.Got it.